package.json

```diff
@@ -200,7 +202,7 @@
     "@typescript-eslint/parser": "^5.27.0",
     "arg": "^5.0.0",
     "babel-eslint": "10.1.0",
-    "babel-jest": "^29.3.1",
+    "babel-jest": "^27.0.6",
```

Comment:
Also, seeing we downgrade a library isn't quite pleasant unless there's a good reason for that.

Reply:
fixed